CMakeLists.txt

```diff
@@ -50,6 +50,13 @@ else()
 endif()
 message(STATUS "Build shared libs: ${BUILD_SHARED_LIBS}")
 
+include(GNUInstallDirs)
+set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}")
```

Comment:
are these overrideable?

Reply:
yes you can override thorugh command line: 

`-DCMAKE_INSTALL_PREFIX=/new/path` and same for the others